src/chains/ethereum/transaction/src/transaction-receipt.ts

```diff
@@ -136,7 +136,7 @@ export class InternalTransactionReceipt {
   public toJSON(
     block: {
       hash(): Data;
-      header: { number: Quantity; baseFeePerGas?: Quantity };
+      header: { number: Quantity };
```

Comment:
Since we no longer need `baseFeePerGas` we should change the receipt's `toJSON` method to not require the `block`, as the `transaction` property already has `blockHash` and `blockNumber` properties we can use.

Reply:
Removed!